guide/locations/_AWS.md

```diff
@@ -42,6 +42,21 @@ Below are examples of configuration options that use values specific to AWS EC2:
   [security groups](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html).
   For example, `securityGroups: mygroup1` or `securityGroups: [ mygroup1, mygroup2 ]`.
 
+### EBS boot volume size
+
+The size of the EBS boot volume for a VM can be specified using `mapNewVolumeToDeviceName` in `templateOptions` as shown below:
+
+    location:
+      jclouds:aws-ec2:
+        region: ...
+        templateOptions:
+          mapNewVolumeToDeviceName:
+           - /dev/sda1
+           - 123
+           - true
```

Comment:
Looks like the indentation is wrong here: should be either on the same level as`mapNewVolumeToDeviceName` or down by 2 spaces

Reply:
The indentation here is correct